### PR TITLE
takeUntil(_: Completable)

### PR DESCRIPTION
**Pull request by MortyMerr**
This was discussed here https://github.com/RxSwiftCommunity/RxSwiftExt/issues/187
Please feel free to notify me, if CHANGELOG or playground updates need, I am ready to handle it

### CODE REVIEW

File: RxSwift/Observables/TakeUntil.swift [hunk 1]
```diff
@@ -225,3 +251,65 @@ final private class TakeUntilPredicate<Element>: Producer<Element> {
...
+
+    func on(_ event: Event<Element>) {
+        self.synchronizedOn(event)
+    }
+
+    func _synchronized_on(_ event: Event<Element>) {
+        switch event {
+        case .next:
+            self.forwardOn(event)
+        case .error:
```
Comment (LucianoPAlmeida) [inline RxSwift/Observables/TakeUntil.swift]: Since the code is the same for both cases, would be an option to use `case .error, .completed:`?

Comment (MortyMerr) [reply RxSwift/Observables/TakeUntil.swift]: Thanks, fixed

**Comment by freak4pc** [comment]
Rethinking this - can anyone find an implementation similar to this in a different ReactiveX implementation? I couldn't find one in RxJava but might've missed something.

**Comment by MortyMerr** [comment]
The main reason for this PR was this issue -  RxSwiftCommunity/RxSwiftExt#187. I didn't check this operator in other ReactiveX implementations...

**Comment by MortyMerr** [comment]
@freak4pc but I think it's a useful operator. Is the absence of it in other Rx repos is a blocker?

**Comment by freak4pc** [comment]
The prerequisite for adding anything into this repo is getting substantial community members providing samples of how they'd use it or if it's needed. It also has a different meaning - `takeUntil(observable)` takes until emission, while for Completable it would be a completion event. Which I'm worried might confused some people. 

We'll need to get some more community interest before we can seriously discuss adding an operator that has no precedence in other Rx implementation.

**Comment by MortyMerr** [comment]
Ok, agree. Just to clarify - PR was open just because you mentioned in RxSwiftCommunity/RxSwiftExt#187 that it might be a good idea :)

**Comment by freak4pc** [comment]
Yeah that's why I wrote "Rethinking this" :D

**Comment by freak4pc** [comment]
Closing this due to lack of activity / community support. If you still want to pursue this feature, let me know and I can reopen, just trying to clean older PRs.

**Comment by LucianoPAlmeida** [comment]
@freak4pc I don't know if it was already discussed, but seeing you closed PR's due inactive make me wonder if we should consider adding the [stale bot](https://github.com/apps/stale).
I've seen it in many open source projects... maybe is useful here too :)
Just an idea :))

**Pull request closed without merging**